### ISSUE TITLE
sge_logs

### ISSUE THREAD

**Issue by Imoteph** [PacificBiosciences/FALCON#317]
Hi

Just notice that the sge_log folder has already a quite big memory footprint even for the small ecoli example. Maybe, it would be advisable to compress the logs after a job is finished or to silence the log by user choice i.e. by redirecting the outputstream to /dev/null.

**Comment by pb-cdunn** [comment]
We might be logging too much. Or maybe we need to use different default log settings. What are typical lines of logging that seem to take the most disk-space?

**Comment by Imoteph** [comment]
I checked my backup files, as I had to remove the logs, and it looks like the detailed output messages of dealigner is mostly the source for exploding the log files. For my purpose I will remove the -v flag as long the pipeline is running through. But it would be a good idea to choose between different log settings.

**Comment by pb-cdunn** [comment]
I don't think daligner prints very much. E.g.
```
Building index for raw_reads.1

 Kshift=30
 BSHIFT=8
 TooFrequent=2147483647
 (Kshift-1)/BSHIFT + (TooFrequent < INT32_MAX)=3
 sizeof(KmerPos)=16
 nreads=229920
 Kmer=15
 block->reads[nreads].boff=313885133
 kmers=310436333
 sizeof(KmerPos)*(kmers+1)=4966981344
 Allocated 310436334 of 16 (4966981344 bytes) at 0x2b0574785010
   Kmer count = 310,436,333
   Using 9.25Gb of space
   Index occupies 4.63Gb

Comparing raw_reads.1 to raw_reads.1

   Capping mutual k-mer matches over 325 (effectively -t18)
   Hit count = 370,344,137
   Highwater of 15.66Gb space

     370,344,137 15-mers (3.764441e-09 of matrix)
         114,960 seed hits (1.168535e-12 of matrix)
         129,957 confirmed hits (1.320975e-12 of matrix)

Building index for c(raw_reads.1)

 Kshift=30
 BSHIFT=8
 TooFrequent=2147483647
 (Kshift-1)/BSHIFT + (TooFrequent < INT32_MAX)=3
 sizeof(KmerPos)=16
 nreads=229920
 Kmer=15
 block->reads[nreads].boff=313885133
 kmers=310436333
 sizeof(KmerPos)*(kmers+1)=4966981344
 Allocated 310436334 of 16 (4966981344 bytes) at 0x2b07c4949010
   Kmer count = 310,436,333
   Using 9.25Gb of space
   Index occupies 4.63Gb

Comparing raw_reads.1 to c(raw_reads.1)

   Capping mutual k-mer matches over 601 (effectively -t24)
   Hit count = 373,735,519
   Highwater of 15.76Gb space

     373,735,519 15-mers (3.798913e-09 of matrix)
         156,952 seed hits (1.595372e-12 of matrix)
         168,518 confirmed hits (1.712937e-12 of matrix)
```
It prints that once for each of the N*N comparisons. Is the problem because you have a *large number* of blocks, and therefore of daligner jobs? Removing `-v` from daligner will not solve that problem.

Possibly you have an old version of the code which accidentally including a repeating log message. If you can be more specific, please re-open. (And in general, when you post an Issue in GitHub, you can help by telling us the SHA1s. Try `git rev-parse HEAD`.)

**Comment by pb-jchin** [comment]
Just a comment, the daligner output is actually really useful.  If the seed hits number is hight but the confirmed number is low, you know the initial k-mer parameters are too sensitive and wasted a lot of CPU cycles for fruitless confirmation calculation.  In the mean time, if the genome size is know or can be estimated, you can see if the confirmed hits agrees with the expected overlap numbers.  For repetitive genomes, those number are good hints to see if the parameters are too aggressive or too loose.

**Comment by Imoteph** [comment]
@pb-cdunn was already on the right track.
Somehow I messed up with my local DALIGNER repo and got a buggy version! After clearing out and reloading, the sge_logs where not as big as at the before. 

@pb-jchin thanks for your comment. We talked at the PAG and you mention you had some good tips how to assemble big plant genomes using falcon. I found some information at #308  and was asking myself if these where the parameters you mention?
